src/components/organisms/Header/constants.tsx

```diff
@@ -25,7 +25,7 @@ export const PRODUCTION_LINKS = [
         text: 'Points',
         link: '/points',
         dataCy: 'points',
-        className: 'flex laptop:hidden',
+        className: 'flex laptop:hidden desktop:flex',
```

Comment:
is this link visible on mobile and tablet?

Reply:
shouldn't this be hidden desktop:flex?

Reply:
I need to show it in mobile, tablet too, because its in mobile menu 

<img width="711" alt="Screenshot 2024-11-04 at 3 23 49 PM" src="https://github.com/user-attachments/assets/7d071558-1afe-4bec-951f-d41a249a2304">